.travis.yml

```diff
@@ -17,6 +17,8 @@ matrix:
   include:
       - python: 2.7
         env: INSTRUMENTAL=yes
+        dist: bionic
+        sudo: true
```

Comment:
This does not do anything in current travis (see the "View config" -> "Config validation" in the travis build)

Reply:
yes, "sudo: true" is supposed to not do anything, but it breaks when i don't use it... so I keep it

Reply:
I wasn't getting the distribution versions and thus python versions I expected from configuration, that being said I tried it some time ago...
Something to try when travis build queue isn't measured in hours...